tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -1349,6 +1350,12 @@ public function asFileOrFolderShouldExist(
 			$this->isEtagValid(),
 			"$entry '$path' expected to exist but not found"
 		);
+		$isCollection = $this->getResponseXmlObject()->xpath("//d:prop/d:resourcetype/d:collection");
+		if ($entry === "folder") {
+			Assert::assertEquals(\count($isCollection), 1, "Unexpectedly, `$path` is not a folder");
+		} else {
```

Comment:
```suggestion
		} elseif ($entry === "file") {
```

We should only make this assert when the step specifically said `file "abc" should exist'

If the feature file has a step `entry "abc" should exist` then we only care that the entry exists. It can be a file or a folder.

Reply:
fixed